### DISCUSSION TITLE
What about `ReactRouterConfig` config from vite?

### DISCUSSION THREAD

**Discussion by zdunecki** [remix-run/react-router#12391] [Q&A]
Hi,

Thank you team for a great work regarding an official version of rr7!

Unfortunately along with a new upgrade a `ReactRouterConfig` disappear from vite setup and now it's replaced by `react-router.config.ts`. 

Seems like a good change for most of folks but does not fit my current needs - the old one via vite was something what I need. 

Do you have plans to support this or any other solution that will cover a use case where I'm trying to built something on top of remix and vite and I'm creating a setup via vite?

**Comment by remorses** [comment]
What is the reason behind this change? I preferred configuring in the vite plugin too

**Reply by sergiodxa** [reply]
Here's the decision docs on why they did that https://github.com/remix-run/react-router/blob/main/decisions/0013-react-router-config-ts.md

**Reply by zdunecki** [reply]
I'ts a good change for most of people and right for a Remix team but now I'm thinking how other would benefit from previous setup. 

If I'm building something on top of vite and react-router how can I setup that if my config is very dynamic and I'm encapsulating almost everything from the end user.

I'm just thinking about an official statement what people with that needs needs to do.
